README.md

```diff
@@ -19,31 +19,67 @@ _Xamarin Forms provides cross-platform APIs, but this project are only worked on
-- CircleListView : A subclass of Xamarin.Forms.ListView control fits in the circular screen and has the circular scrollbar.
-- CirclePage : A subclass of Xamarin.Forms.Page that can show Circular ProgressBar and Circular Slider and the button on the bottom of the screen. it can also show MenuItems on the circular menu.(MoreOption)
-- CircleProgressBarSurfaceItem : A control fits in the circular screen. it can be shown in CirclePage only.
-- CircleSliderSurfaceItem : A Slider control that responds to the bezel action and fits in the circular screen.
-- CircleScrollView : A subclass of Xamarin.Forms.ScrollView that can be scrolled by the bezel action.
-- CircleStackLayout : A container to layout children linear in the circular area.
-- CircleStepper : A control to select a number of steps fits in the circular screen.
-- CircleSurfaceEffectBehavior : The CircleSurfaceEffectBehavior is an effect which allows you to insert views that require CircleSurface.
-- ContextPopupEffectBehavior : the behavior to show a small popup has one or two button sticky with any control.
-- IndexPage : A subclass of multiple pages that can slide child page horizontal and has dots on top of the screen for the number of child pages.
-- InformationPopup : A popup has a control to show progress and one button on the bottom side of the circular screen.
-- IRotationReceiver : A event receiver for using the bezel action.
-- Radio : A radio control.
-- GoogleMapView : A GoogleMapView is used to display google javascript map.
-- MediaView : A MediaView is used to display a video on the screen.
-- PopupEntry : The PopupEntry is a class that extends Xamarin.Forms.Entry. It makes a new layer when editing text on the entry.
-- Toast : A popup for simple feedback.
-- TwoButtonPopup : A popup has two buttons on the left and right side of the circular screen.
-- TwoButtonPage : A subclass of Xamarin.Forms.Page has two buttons on the left and right side of the circular screen.
-
-![widgets](doc/design/data/widgets.png)
+The Tizen Wearable CircularUI project provides UI controls to fits circular screen and to interact to user using hardware feature provided from Samsung Gear such as bezel rotation.
+It provides newly created controls for Tizen Wearable.
+
+- [CircleDateTimeSelector](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleDateTimeSelector.html) : A View to select date or time that fits in the circular screen. Unlike Xamarin.Forms.DatePicker(TimePicker) it can be shown on the page without popup.
+- [CircleImage](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleImage.html) : An Image that crops an image in a circle shape.
+- [CircleProgressBarSurfaceItem](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleProgressBarSurfaceItem.html) : A progress control that fits in the circular screen. it can be shown in CirclePage only.
```

Comment:
it can be shown in CirclePage only. -> This control can be only used in the CircleSurfaceView.